src/panels/lovelace/editor/view-editor/hui-dialog-edit-view.ts

```diff
@@ -212,6 +235,15 @@ export class HuiDialogEditView extends LitElement {
                 .path=${mdiPlaylistEdit}
               ></ha-svg-icon>
             </ha-list-item>
+            <mwc-list-item graphic="icon">
+              ${this.hass!.localize(
+                "ui.panel.lovelace.editor.edit_view.move_to_dashboard"
+              )}
+              <ha-svg-icon
+                slot="graphic"
+                .path=${mdiFileMoveOutline}
+              ></ha-svg-icon>
+            </mwc-list-item>
```

Comment:
Due to my pr getting in, there's now a bit of an aligment issue since the first list item uses ha-list-item and the second one is a regular mwc-list-item.

Reply:
:+1:  I'll fix it